README.md

```diff
@@ -1,4 +1,27 @@
-# MongoDB Documentation Jupyter Notebooks
+# MongoDB Documentation Notebooks
 
-This repo contains Jupyter Notebooks that follow the
-[MongoDB Vector Search documentation](https://www.mongodb.com/docs/atlas/atlas-vector-search/vector-search-overview/).
+This repository contains Jupyter Notebooks that follow
+tutorials and code examples in MongoDB's official [Atlas Vector Search documentation](https://www.mongodb.com/docs/atlas/atlas-vector-search/vector-search-overview/). You can run, download, and modify these notebooks as you learn how to use MongoDB Atlas Vector Search for your use case.
+
+## Overview
+
+Each notebook corresponds to a page or example in our documentation.
+Refer to the docs page linked in each notebook for prerequisites, set-up instructions, and detailed explanations of the code.
+
+The following table summarizes the contents of the notebooks in each directory:
+
+| Directory          | Description                                      |
+|--------------------|--------------------------------------------------|
+| [/create-embeddings](https://github.com/mongodb/docs-notebooks/tree/main/create-embeddings) | Learn how to generate embeddings for vector search |
+| [/get-started](https://github.com/mongodb/docs-notebooks/tree/main/get-started) | Complete our quick start tutorial |
+| [/integrations](https://github.com/mongodb/docs-notebooks/tree/main/integrations) | Implement RAG with popular frameworks that integrate with MongoDB |
+| [/manage-indexes](https://github.com/mongodb/docs-notebooks/tree/main/manage-indexes) | Create, view, edit, and delete  vector search indexes |
```

Comment:
Tiny nit - just happened to spot an extra space here!
```suggestion
| [/manage-indexes](https://github.com/mongodb/docs-notebooks/tree/main/manage-indexes) | Create, view, edit, and delete vector search indexes |
```

Reply:
good eye!